index.bs

```diff
@@ -3754,6 +3779,39 @@ extension for transaction authorization.
     });
 </pre>
 
+## Aborting Authentication Operations ## {#sample-aborting}
+
+The below example showcases how a developer may use the AbortSignal parameter to abort an
+ongoing credential registration operation. Similiar procedure applies to an ongoing
+authentication operation. 
+
+<pre class="example" highlight="js">
+    const authAbortController = new AbortController(); 
+    const authAbortSignal = controller.signal;
+
+    var options = {
+        // A list  of options. 
+    }
+
+    navigator.credentials.create({"publicKey": options}, authAbortSignal)
```

Comment:
IIUC, we were contemplating passing the signal as part of the `options`, rather than passing`authAbortSignal` as a separate arg?  

see also: https://github.com/w3c/webauthn/pull/544/files#r146071432

Reply:
This pull request demonstrates what it would look like to keep this in a separate arg. I intend to submit another pull request that shows what it'd look like to keep the arg inside option. 

The main argument for keeping it as a separate arg is that in the future when we allow multiple credentials to be requested in one operation, developers can abort them altogether. For example: 

    
    
    const authAbortController = new AbortController() 
    const authAbortSignal = controller.signal

    var publicKeyOptions = { // A list of public key related options.}
    var passwordOptions = { // A list of password related options.}
    var federatedOptions = { // A list of federated options.}

    navigator.credentials.get({
         {"publicKey": publicKeyOptions, 
          "password": passwordOptions,
          "federated": federatedOptions,
         }, authAbortSignal)
        .then(function (attestation) {
            // Register the user. 
        }).catch(function (error) {
            if (error = "AbortError") {
                // Let the server know a key hasn't been created. 
            }
        });

    // Assume widget shows up whenever auth occurs. 
    if (widget = "disappear") {
        authAbortSignal.abort(); 

        authAbortSignal.onabort = function () {
            // Migrate away from the page and other necessary clean-ups. 
        }
    }
`
I was convinced after discussion with @jyasskin. @jyasskin do you have more to add?